.github/workflows/deploy.yaml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 on:
+  workflow_dispatch:
```

Comment:
While it's useful to be able to run the quality checks on an arbitrary branch, I don't think we want to be able to publish the docs from one!

Reply:
ah, true, I automatically added it to both workflows, without really thinking what the workflow is :)